src/content/nl/2020/security.md

```diff
@@ -0,0 +1,1026 @@
+
+## Aanvallen verijdelen
+
+Moderne webapplicaties worden geconfronteerd met een grote verscheidenheid aan beveiligingsrisico's. Het onjuist coderen of opschonen van gebruikersinvoer kan bijvoorbeeld resulteren in een cross-site scripting (XSS) -kwetsbaarheid, een klasse van problemen die webontwikkelaars al meer dan een decennium lastig vallen. Naarmate webapplicaties steeds complexer worden en nieuwe soorten aanvallen worden ontdekt, doemen er nog meer bedreigingen op. <a hreflang="en" href="https://xsleaks.dev/">XS-Leaks</a>, bijvoorbeeld, is een nieuwe klasse van aanvallen die tot doel heeft gebruik te maken van de gebruikersspecifieke dynamische reacties die webapplicaties teruggeven.
+
+Als een webmailcliënt bijvoorbeeld een zoekfunctionaliteit biedt, kan een aanvaller verzoeken voor verschillende trefwoorden activeren en vervolgens via verschillende zijkanalen proberen vast te stellen of een van deze trefwoorden resultaten heeft opgeleverd. Dit biedt de aanvaller in feite een zoekmogelijkheid in de mailbox van een onwetende bezoeker op de website van de aanvaller.
+
+Gelukkig bieden webbrowsers ook een groot aantal beveiligingsmechanismen die zeer effectief zijn om de gevolgen van een mogelijke aanval te beperken, bijv. via de `script-src` richtlijn van CSP kan een XSS-kwetsbaarheid erg moeilijk of onmogelijk worden om te misbruiken.
+
+Sommige andere beveiligingsmechanismen zijn zelfs nodig om bepaalde soorten aanvallen te voorkomen: om <a hreflang="en" href="https://en.wikipedia.org/wiki/Clickjacking">clickjacking</a> aanvallen te voorkomen, moet ofwel de `X-Frame-Options` header aanwezig zijn, of als alternatief kan de `frame-ancestors` richtlijn van CSP worden gebruikt om vertrouwde partijen aan te duiden die het huidige document kunnen insluiten.
+
+{{ figure_markup(
+  image="security-adoption-of-security-headers.png",
+  caption="Goedkeuring van beveiligingskoppen",
+  description="Staafdiagram met de prevalentie van verschillende beveiligingskoppen voor mobiele pagina's in 2019 en 2020. <span lang="en">`Content-Security-Policy`</span> is 5% op desktop en 11% op mobiel, `Expect-CT` is 8% op desktop en 11% op mobiel, <span lang="en">`Feature-Policy`</span> is 0% op desktop en 1% op mobiel, <span lang="en">`Referrer-Policy`</span> is 6% op desktop en 7% op mobiel, <span lang="en">`Report-To`</span> is 2% op desktop en 3% op mobiel, <span lang="en">`Strict-Transport-Security`</span> is 13% op desktop en 17% op mobiel, <span lang="en">`X-Content-Type-Options`</span> is 26% op desktop en 30% op mobiel, `X-Frame-Options` is 24% op desktop en 27% op mobiel, en `X-XSS-Protection` is 16% op desktop en 18% op mobiel.",
```

Comment:
I don't think you need the `lang="en"` around technical terms. Removing them also resolves your nested quote issue stopping the chapter generation:

```suggestion
  description="Staafdiagram met de prevalentie van verschillende beveiligingskoppen voor mobiele pagina's in 2019 en 2020. `Content-Security-Policy` is 5% op desktop en 11% op mobiel, `Expect-CT` is 8% op desktop en 11% op mobiel, `Feature-Policy` is 0% op desktop en 1% op mobiel, `Referrer-Policy` is 6% op desktop en 7% op mobiel, `Report-To` is 2% op desktop en 3% op mobiel, `Strict-Transport-Security` is 13% op desktop en 17% op mobiel, `X-Content-Type-Options` is 26% op desktop en 30% op mobiel, `X-Frame-Options` is 24% op desktop en 27% op mobiel, en `X-XSS-Protection` is 16% op desktop en 18% op mobiel.",
```

Alternatively you could escape the quotes.